pages/linux/sslstrip.md

```diff
@@ -0,0 +1,20 @@
+# sslstrip
+
+> The sslstrip is a tool that transparently hijacks HTTP traffic on a network.
+> More information: <https://www.kali.org/tools/sslstrip/>.
```

Comment:
```suggestion
> More information: <https://github.com/L1ghtn1ng/sslstrip>.
```

Kali's link is not the official link for this project.

Reply:
@jimmychang851129 Using kali webpage breaks how we source more information. 

See https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#more-information-links for details.

> On the More information link line, we prefer linking to the author's provided documentation of the command line reference or the man page. When not available, use https://manned.org/ as the default fallback for all platforms (except osx and BSD platforms other than FreeBSD). Alternatively, you can link to the author's website or a tutorial page if the command doesn't have a documentation page.

Reply:
IMO we should use Kali's documentation, the priority is the most complete resource, and not if it is the official one or not, that's why we use man pages sometimes.

It's kind of an informal rule, we should indeed document it.

Reply:
Also, when there is no documentation in the README, the style guide makes it ver clear:

> Alternatively, you can link to the author's website or a tutorial page if the command doesn't have a documentation page.

IMO a README without CLI usage can't be considered a documentation in this case.

Reply:
The author's github page contains a README (agreed it is not comprehensive) mentioning the important prerequisites needed to run the command which is missing in the Kali's link.

@jimmychang851129 Are you able to confirm if you were able to run the tool without the prerequisites mentioned in https://github.com/L1ghtn1ng/sslstrip/blob/master/README.md ?

Reply:
```suggestion
> More information: <https://manned.org/sslstrip.1>.
```

Manned has the [same Documentation](https://manned.org/sslstrip.1) but for an older version with no change in commands, we can link to it or to the Kali page instead of the author's page for this special case (as the author's docs don't have any documentation for [this file](https://github.com/L1ghtn1ng/sslstrip/blob/master/sslstrip.py)).

IMO as @vitorhcl says, Kali docs might be a better reference for this page.

---

Maybe, I will open an issue/PR to the author's repo for requesting/adding docs to README (referencing this PR).

Reply:
In case it got overlooked, note that Kali page do not contain the **PREREQUISITES** for running the command.

Reply:
@fazlearefin  
I agree with the **prerequisites part** .  sslstrip is merely for deciphering Internet packet, and the prerequisite is required because users should intercept the Internet packet before deciphering it.

The idea is like, users should have a sql server running to have sqlmap to work. 
In short, the prerequisite is a must but is not related to the tool iteself.
Perhaps, there should be some way to inform users the prerequisite through tldr.  Any suggested format on this?

Reply:
IMO,
Since tldr is not a penetrating tools manuals, the prerequisite (i.e. steps to intercept Internet packets) is not necessary to be shown on tldr, but I will edit the description to at least mention the prerequisite part.

Thanks a lot for brining up this, mentioning the prerequisite will be beneficial for the users.